modules/virt-about-hostpath-provisioner.adoc

```diff
@@ -6,22 +6,9 @@
 [id="virt-about-hostpath-provisioner_{context}"]
 = About the hostpath provisioner
 
-The hostpath provisioner is a local storage provisioner designed for
-{VirtProductName}. If you want to configure local storage for
-virtual machines, you must enable the hostpath provisioner first.
+When you install the {VirtProductName} Operator, the hostpath provisioner (HPP) Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName}. If you want to use the HPP, you must:
 
-When you install the {VirtProductName} Operator, the hostpath provisioner Operator
-is automatically installed. To use it, you must:
+* Create the HPP custom resource.
```

Comment:
Maybe let's keep 'HostPathProvisioner' because it is what is expected to be in the yaml's 'kind', so we'll not confuse the users with acronyms at this point :)